service/src/test/java/greencity/mapping/TagVOMapperTest.java

```diff
@@ -0,0 +1,101 @@
+package greencity.mapping;
+
+import greencity.dto.language.LanguageVO;
+import greencity.dto.tag.TagTranslationVO;
+import greencity.dto.tag.TagVO;
+import greencity.entity.Language;
+import greencity.entity.Tag;
+import greencity.entity.localization.TagTranslation;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static greencity.ModelUtils.getTag;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class TagVOMapperTest {
+    private final TagVOMapper mapper = new TagVOMapper();
+
+    @Test
+    public void convert_withCorrectData_isTrue(){
+        var toConvert = getTag();
+        var expected = buldTagVO(toConvert);
+
+        var actual = mapper.convert(toConvert);
+
+        Assertions.assertThat(actual).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @NullSource
+    public void convert_withNullToConvertObject_throwNullPointer(Tag tag){
```

Comment:
As for me test method name - `convert_whenTasIsNull_expectNPE` will suites better (or `expectNullPointerException`, `throwNPE` etc)